### DISCUSSION TITLE
Confused about how Planned Start/Planned End are supposed to work

### DISCUSSION THREAD

**Discussion by lukestein** [cpvalente/ontime#941] [Q&A]
If I create an event running from 11:30 with a duration of 30 minutes, I see "expected start"/"end" as I expected
![image](https://github.com/cpvalente/ontime/assets/44452336/5150e68d-2ec7-4eb8-8a09-65b8e192f103)

However, as soon as I start the event, I get planned start of 1:00 and planned end of 21:00, which I don't at all understand.
![image](https://github.com/cpvalente/ontime/assets/44452336/e8e56259-d9b5-4327-9972-772af5905859)

User error? Misunderstanding? Bug?

**Comment by cpvalente** [comment]
Hi @lukestein , I apologise for the rough start
I need to work on improving documentation to the new elements of the interface, but these should be very self explanatory
Planned start and Planned end are the respective start and end of the rundown as a whole. As you see in the first screenshot

I think this is definitely something we need to look at, would you mind sending us your project file along with some information on your system.
- How are you running Ontime (from docker or install)
- Which device and OS is your browser running on
- Which browser and which version are you using?
- What is your screen resolution?

You can send the project file here or to mail@getontime.no and we will take a look.
You can also find us in the video engineering discord

**Reply by lukestein** [reply]
Running via Docker on a Synology NAS. Browser is Chrome 124.0.6367.156 on a Mac.

I'm not sure this is a documentation problem. This short video hopefully shows what I'm confused about (sound on for my voiceover explanation):

https://github.com/cpvalente/ontime/assets/44452336/5de64dba-2007-4fc3-b7cb-82f2ec2e2e93

**Reply by cpvalente** [reply]
Thank you.
This is definitely an issue on our side, I am investigating as we speak.

The reason why I mention documentation, is that I feel that users should be confident to point this as a bug.
The Planned start and planned end datasets should have little room for nuance.

I noticed some small defects in your first screenshots. Do you mind sharing your screen resolution?

**Reply by lukestein** [reply]
Thanks for digging. Happy to move to a Github issue rather than discussion if it's more useful for you.

I do see some little layout issues (like the link/padlock icons overwriting on the text), but they were in an arbitrarily sized window on my monitor and I don't know the resolution they were set at. I also tried a bit just now but can't recreate the layout defects—though I'm still getting the planned start/planned end issue. I'll keep my eye out on layout as I play around more and add a report with screenshots and resolutions if I can find problems there.
